src/genie/libs/parser/ios/asr901/show_environment.py

```diff
@@ -0,0 +1,170 @@
+# Metaparser
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Or, Optional
+
+# ==============================
+# Schema for 'show environment'
+# ==============================
+class ShowEnvironmentSchema(MetaParser):
+
+    ''' Schema for "show environment" '''
+
+# These are the key-value pairs to add to the parsed dictionary
+    schema = {
+        'PowerSupply': {
+            Any() : {
+                Optional('volt') : str,
+                Optional('status') : str
+            }
+        },
+        'Fan': {
+            Any() : {
+                Optional('status') : str,
+                Optional('speed') : str
+            }
+        },
+        'Sensor' : {
```

Comment:
same. and suggest alternative key name on other comment
```suggestion
        'board_temperature' : {
```

Reply:
confirmed the change.